planner/property/physical_property.go

```diff
@@ -51,8 +53,25 @@ const (
 	BroadcastType
 	// HashType requires current task to shuffle its data according to some columns.
 	HashType
+	// CollectType requires all the task pass the data to one node (tidb/tiflash).
+	CollectType
 )
 
+// ToExchangeType generates ExchangeType from MPPPartitionType
+func (t MPPPartitionType) ToExchangeType() tipb.ExchangeType {
+	switch t {
+	case BroadcastType:
+		return tipb.ExchangeType_Broadcast
+	case HashType:
+		return tipb.ExchangeType_Hash
+	case CollectType:
+		return tipb.ExchangeType_PassThrough
```

Comment:
???

Reply:
currently, all the passthrough exchange is actually collectType

Reply:
renamed CollectType to PassThroughType.